.coveragerc

```
@@ -0,0 +1,16 @@
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Don't complain if non-runnable code isn't run:
+    if 0:
+    if __name__ == .__main__.:
+
+    # pragma: no cover
+    # pragma: no branch
```

```suggestion
    .*# pragma: no cover
    .*# pragma: no branch
```
Since I don't know exactly how these regexes are used, I don't know if these additions makes any difference or not.  But they match what I have, which may have been copied or left as is.  They do make clear that a line with anything followed by these comments should match and be excluded.